tests/crashes/110630.rs

```diff
@@ -1,28 +0,0 @@
-//@ known-bug: #110630
```

Comment:
I opted to simply remove these GCE crashes since they don't really capture anything interesting in the type system, and also GCE is an abandoned feature.